package.json

```diff
@@ -42,19 +42,16 @@
     "babel-loader": "8.2.3",
```

Comment:
Check if we still need `@babel/core, babel-loader, postcss` packages.

Reply:
@babel/core, babel-loader required for storybook. postcss required for autoprefixer - removed both.